### PR TITLE
Opens account

### PR DESCRIPTION
**Pull request by aaalata**
Second scenario - opening an account.

### CODE REVIEW

File: src/test/java/pl/edu/agh/iet/katabank/steps/AccountManagementSteps.java [hunk 1]
```diff
@@ -46,5 +42,22 @@ public AccountManagementSteps() {
                 () -> assertThat(customerAccounts)
                         .containsExactlyInAnyOrder(firstAccount, secondAccount));
 
+        Given("^a customer wants to open an account$", () -> {
+            customer = new Customer();
+        });
```
Comment (anthonyflea) [inline src/test/java/pl/edu/agh/iet/katabank/steps/AccountManagementSteps.java]: (Small styling comment)
Replace with lambda expression instead lambda statement. Intellij should suggest that.

File: src/test/java/pl/edu/agh/iet/katabank/steps/AccountManagementSteps.java [hunk 2]
```diff
@@ -46,5 +42,22 @@ public AccountManagementSteps() {
...
+            firstAccount = new Account(customer);
+            accountsRepository.addAccount(firstAccount);
+        });
+
+        Then("^there is a new account on his account list$", () -> {
+            customerAccounts = bank.getAccountsForCustomer(customer);
+            assertThat(customerAccounts).contains(firstAccount);
+        });
+        And("^the balance on this account is (\\d+)$", (Integer arg0) -> {
+            assertThat(firstAccount.getBalance()).isEqualByComparingTo("0");
```
Comment (anthonyflea) [inline src/test/java/pl/edu/agh/iet/katabank/steps/AccountManagementSteps.java]: Give argument "arg0" more meaningful name.
Compare account balance with BigDecimal of value 0 rather then String.

**Review by anthonyflea — CHANGES_REQUESTED** [review]
I reviewed only your commits. Some comments added.

File: src/test/java/pl/edu/agh/iet/katabank/steps/AccountManagementSteps.java (hunk 1 above)
Comment (aaalata) [reply src/test/java/pl/edu/agh/iet/katabank/steps/AccountManagementSteps.java]: It does suggest indeed. Corrected.

File: src/test/java/pl/edu/agh/iet/katabank/steps/AccountManagementSteps.java (hunk 2 above)
Comment (aaalata) [reply src/test/java/pl/edu/agh/iet/katabank/steps/AccountManagementSteps.java]: 100% right. Corrected.

**Review by aaalata — COMMENTED** [review]
Two suggested changes approved&corrected.
No changes skipped.
Thanks!

**Review by anthonyflea — APPROVED** [review]

**Pull request merged**